run-tests.pl

```diff
@@ -177,7 +182,10 @@ sub usage
    -n, --no-tls                 - prefer plaintext client connections where
                                   possible
 
-       --bind-host HOST         - when starting listeners (eg homeservers or
```

Comment:
this was deliberate. the `--long-options` all line up.

Reply:
Oh whoops. I'll fix.